client/webserver/webserver_test.go

```diff
@@ -147,11 +146,33 @@ func (r *TReader) Read(p []byte) (n int, err error) {
 func (r *TReader) Close() error { return nil }
 
 type TConn struct {
-	msg []byte
+	msg       []byte
+	reads     [][]byte      // data for ReadMessage
+	respReady chan []byte   // signal from WriteMessage
+	close     chan struct{} // Close tells ReadMessage to return with error
```

Comment:
I see this is read in `ReadMessage`, but it doesn't look like it's ever actually closed. Am I reading that right?

Reply:
Ahhh, I suppose the intent was to put that in `(*TConn).Close`